pagopa/pn-frontend
# PN-11746 - Handle multi-language on PG

## Short description
Handle multi-language on PG and keep the selected language on refresh page.
This PR resolve: [PN-11751](https://pagopa.atlassian.net/browse/PN-11751), [PN-11752](https://pagopa.atlassian.net/browse/PN-11752) and [PN-11754](https://pagopa.atlassian.net/browse/PN-11754)

## List of changes proposed in this pull request
- Get language from token-exchange URLs from Selfcare
- Store lang on sessionStorage
- Add lang params to external URLs

## How to test
On PG, use a direct access link and append a lang query parameter with a language (e.g. `&lang=en`) to it. Verify that the page is translated into the selected language. Also, try refreshing the page and check that the selected language is retained

[PN-11751]: https://pagopa.atlassian.net/browse/PN-11751?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[PN-11752]: https://pagopa.atlassian.net/browse/PN-11752?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[PN-11754]: https://pagopa.atlassian.net/browse/PN-11754?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
